dev/com.ibm.ws.jpa.container.core/resources/com/ibm/ws/jpa/jpa.nlsprops

```diff
@@ -312,6 +312,11 @@ EMPTY_PROP_NAME_VALUE_CWWJP0056W=CWWJP0056W: A JPA default property configured i
 EMPTY_PROP_NAME_VALUE_CWWJP0056W.explanation=The name and value attributes for a JPA default property must not be empty.
 EMPTY_PROP_NAME_VALUE_CWWJP0056W.useraction=Change the server configuration to provide a non-empty name and value for the property or remove the property.
 
+# 0=property name, 1=property value
+JDBC_PROP_NAME_CWWJP0057W=CWWJP0057W: A JPA default property configured in the server.xml file is not recommended to be used here: name = [{0}], value = [{1}].
```

Comment:
- in general, we're advised to avoid making recommendations- can we instead just tell them not to use it?
- what does it mean that a "default" property is configured? Isn't the default what you get when you don't specify configuration? Also, the message calls it a "JPA default property" and the explanation says "JDBC property". If they're the same thing it's better to use one consistent term.

suggested edit:
 The following JDBC property, which is configured in the server.xml file, is not intended for Java or Jakarta EE applications: name = [{0}], value = [{1}].

Reply:
> what does it mean that a "default" property is configured?

Users configure "default" JPA properties in their server.xml configuration. These default properties become the defaults for all  container managed persistence units in all applications running on that server

Here is an example of a JPA JDBC property:
```xml
<server>
    <featureManager>
        ...
    </featureManager>

    <jpa>
        <defaultProperties>
            <property name="javax.persistence.jdbc.url" value="jdbc:mysql://localhost/testDB" />
        </defaultProperties>
    </jpa>
</server>
```
If a customer uses one of the `javax.persistence.jdbc.*` properties (referred to as JDBC properties), then we will log warning `CWWJP0057W` to inform them that this property should not be used in this way.

> we're advised to avoid making recommendations- can we instead just tell them not to use it?
Ill have to discuss with my team if we can ignore these properties instead of just warn against their use. If so, I can update the message accordingly